crypto/poly1305/asm/poly1305-x86_64.pl

```diff
@@ -344,15 +348,18 @@ sub poly1305_iteration {
 .type	__poly1305_block,\@abi-omnipotent
 .align	32
 __poly1305_block:
+.cfi_startproc
 ___
 	&poly1305_iteration();
 $code.=<<___;
 	ret
+.cfi_endproc
 .size	__poly1305_block,.-__poly1305_block
 
 .type	__poly1305_init_avx,\@abi-omnipotent
 .align	32
 __poly1305_init_avx:
+.cfi_startproc
 	mov	$r0,$h0
 	mov	$r1,$h1
 	xor	$h2,$h2
```

Comment:
Does this function need to CFA?

Reply:
Doesn't look like it.

Reply:
It is called from a CFA instrumented function, so the runtime stack walking
will not be able to continue the stack walk.

Reply:
I think that since you start a new CFI, the CFA defaults to the stack pointer again.

Reply:
Right, and gdb's bt command works just fine,
but runtime-stack walks don't, and tools using that
for performance/coverage metrics don't work right,
when that .cfi is missing, the runtime-stack walk
assumes it can't unwind beyond this frame.